### DISCUSSION TITLE
Unable to run pipeline from GOCD. It throws ssh related error

### DISCUSSION THREAD

**Discussion by Naveenkumar453** [learndevopsonline/learndevopsonline#1898] [Q&A]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/55714376/933c7f80-6154-4852-b9f4-1a36c0c77ee9)

As mentioned in the screenshot, from pipe line command is not running but when run manually its working and I am able to ping mysql machine from ansible-control machine also.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/55714376/78d4150c-20df-4e36-82f8-02d054055954)

I have copied ssh keys as well but still it is not working through pipeline.

**Comment by chaithanya1812** [comment]
Hi @Naveenkumar453 , can you send screenshot of encoding SSH_PASSWORD secure variable command,
What you done?..

**Comment by chaithanya1812** [comment]
Hi @Naveenkumar453 ,
##  I hope you utilised the lab session well, finally we found the error & we solved it well...
